build/dev/docker-compose_dss.yml

```diff
@@ -3,65 +3,68 @@
 # To bring up this system, see standalone_instance.md.
 
 services:
-    local-dss-crdb:
-        image: cockroachdb/cockroach:v21.2.7
-        command: start-single-node --insecure
-        ports:
-            - "26257:26257"
-            - "8080:8080"
-        restart: always
-        networks:
-            - dss_sandbox_default_network
 
-    local-dss-rid-bootstrapper:
-        build:
-            context: ../..
-            dockerfile: Dockerfile
-        image: interuss-local/dss
-        volumes:
-            - local-dss-data:/var/local-dss-data
-            - $PWD/startup:/startup:ro
-        entrypoint: /startup/rid_bootstrapper.sh
-        depends_on:
-            - local-dss-crdb
-        networks:
-            - dss_sandbox_default_network
+  local-dss-crdb:
+    image: cockroachdb/cockroach:v21.2.7
+    command: start-single-node --insecure
+    ports:
+      - '26257:26257'
+      - '8080:8080'
+    restart: always
+    networks:
+      - dss_sandbox_default_network
+      
+local-dss-rid-bootstrapper:
```

Comment:
Identation is missing for rid-bootstrapper

Reply:
Done